File: solutions/beecrowd/1154/1154.cpp
#include <cstdint>
#include <cstdio>

int main() {
    float sum = 0.0;
    std::int16_t n, count = 0;

    while (scanf("%d", &n) && n >= 0) {
        sum += n;
        count += 1;
    }
    printf("%.2f\n", sum / count);

    return 0;
}

File: solutions/beecrowd/1154/1154.py
import sys

input = sys.stdin.readline
output = sys.stdout.write

sum = 0
count = 0

while line := input().strip():
    n = int(line)
    if n < 0:
        break

    sum += int(line)
    count += 1

output(f'{sum /count:.2f}\n')
